src/main/java/com/minecolonies/coremod/client/gui/WindowHutDeliveryman.java

```diff
@@ -1,14 +1,15 @@
 package com.minecolonies.coremod.client.gui;
 
 import com.minecolonies.blockout.controls.Button;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.blockout.controls.Button'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)